apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/sql/SqlStore.java

```diff
@@ -0,0 +1,17 @@
+package com.apollographql.android.cache.normalized.sql;
+
+
+import com.apollographql.android.cache.normalized.CacheStore;
+import com.apollographql.android.cache.normalized.Record;
+
+import javax.annotation.Nullable;
+
+public class SqlStore extends CacheStore {
```

Comment:
`RecordSet` is more of a `aggregator` tool used by the normalizer during parsing. I was lazy and used it inside `InMemoryStore` because technically a `RecordSet` is an `InMemoryStore`, just that a `RecordSet` was intended to be used on only one query, so it felt wrong to make it implement `CacheStore`.

Though, depending on how this develops, I might not be opposed to keeping the layer of indirection